src/core/css/SearchResult.scss

```diff
@@ -1,18 +1,27 @@
+@import "~core/css/inc/vars";
+
+$border-radius: 9px;
+$secondary-color: #6a6a6a;
+
 .SearchResult {
-  border: solid #cfcfcf;
-  border-width: 1px 1px 0;
+  background: $list-item-background-color;
+  border-bottom: 6px solid #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
 
-  &:last-of-type {
-    border-bottom-width: 1px;
+  &:first-of-type {
+    border-top-left-radius: $border-radius;
+    border-top-right-radius: $border-radius;
   }
 
-  &:hover {
-    background-color: #efefef;
+  &:last-of-type {
+    border-bottom: 0;
+    border-bottom-left-radius: $border-radius;
+    border-bottom-right-radius: $border-radius;
   }
 }
 
 .SearchResult-link {
-  color: #333;
+  color: $text-color-default;
```

Comment:
There's now `$text-color-default` and `$primary-font-color` which sound like synonyms. Is `$primary-font-color` just for headings? Maybe that should be mentioned in its name.